code/modules/mob/living/carbon/monkey/skeleton.dm

```diff
@@ -8,9 +8,12 @@
 	canWearClothes = 0
 	canWearHats = 0
 	canWearGlasses = 0
+	canWearBack = 0
+	canWearMasks = 0
 	languagetoadd = LANGUAGE_CLATTER
 	brute_damage_modifier = 2
 	movement_speed_modifier = 0.23
+	safe_oxygen_min = 0
```

Comment:
To be fair, skeletons not needing to breath, but their disembodied heads needing to just didn't sit right